### ISSUE TITLE
Applying explicit font size class overridden by component style

### ISSUE THREAD

**Issue by hitendramalviya** [OfficeDev/office-ui-fabric-react#906]
### Bug Report

- __Package version(s)__: 1.3.4
- __Browser and OS versions__: Chrome Windows 10

#### Describe the issue:

Applying class `ms-font-s` to `<body class="ms-font-s">...</body>` but font-size is overridden by style injected by component and always keeping the same font-size i.e. 14px.

**Comment by mdahamiwal** [comment]
You will need a more specific selector by adding more classes. If font-size is the only thing you need to control then adding another class on the container should suffice. 
http://codepen.io/mdahamiwal/pen/egydeL

**Comment by hitendramalviya** [comment]
Yes we can define more specific selector classes but why each component injecting a separate stylesheet into head section. If I already included 
` <link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/1.0/fabric.min.css">
 <link rel="stylesheet" href="https://appsforoffice.microsoft.com/fabric/1.0/fabric.components.min.css">`
into head there is no more required to inject styles for each component. Can we set somewhere not to call style loader.
